WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -1019,6 +1024,74 @@ class MySiteViewModelTest : BaseUnitTest() {
         assertThat(quickActionsBlock).isNotNull
     }
 
+    @Test
+    fun `given QS dynamic cards cards feature is on, when start quick start is triggered, then QS starts`() {
+        whenever(quickStartDynamicCardsFeatureConfig.isEnabled()).thenReturn(true)
+
+        viewModel.startQuickStart(siteId)
+
+        verify(quickStartRepository).startQuickStart(siteId)
+    }
+
+    @Test
+    fun `given no selected site, when start quick start is triggered, then QSP is not shown`() {
+        whenever(quickStartDynamicCardsFeatureConfig.isEnabled()).thenReturn(false)
+        whenever(selectedSiteRepository.getSelectedSite()).thenReturn(null)
+
+        viewModel.startQuickStart(siteId)
+
+        assertThat(navigationActions).isEmpty()
+    }
+
+    @Test
+    fun `given QS is not available for the site, when quick start start is triggered, then QSP is not shown`() {
```

Comment:
Minor: Typo duplicate `start` here and in the following test case can be removed.

Reply:
Thanks for noticing that @ashiagr ! 🙏 

Actually, this was done on purpose, not a typo, although it seems to... 😄 We call the feature `Quick Start` and then we call the function that triggers that `start` + `feature`, so in plain English it becomes `quick start start`. Do you still think that removing the `start` is better?

PS: Same for the below.

Reply:
Ohhh 😯! I didn't realize it 🤦‍♀️. Thanks for clarifying 👍. 

However, I see that you used two different when clauses:

1. `when quick start start is triggered`

2. `when start quick start is triggered`

I like 2. as it doesn't look like a typo 😄, and also makes test statements consistent. Wdyt? Or is that also on purpose 🤔.

Reply:
😄 Oh, I didn't realise I used two different when clauses, thanks for noticing that! 🥇 

Yes, I am going to pick `2` as well and update the test, thanks so much @ashiagr ! 🙏

Reply:
This is now done here: 6b4f8ec1c64bf52b733425db3b2789db7c699400